kuma/users/views.py

```diff
@@ -418,5 +419,11 @@ def get_context_data(self, **kwargs):
         })
         return context
 
+    def dispatch(self, request, *args, **kwargs):
+        ret = verify_honeypot_value(request, None)
```

Comment:
I had `response` in there at first, but then I saw that `verify_honeypot_value` can also return `None`, so I changed it. I.e., the returned value may not be a response, so I just named it `ret`. I could go either way on it though. I'll leave it up to you.